linode/manager
# M3-5699: Add support ticket link to notice text

## Description 📝

**What does this PR do?**

Links the words "open a support ticket" in error notices to the support page with a modal for opening a new support ticket.

The code additions follow the existing error intercept pattern in `request.tsx` that replaces error text by matching on text from the API's error reason. Now, when any error has a reason that contains the text "open a support ticket", we replace it with the SupportError component. The SupportError component displays the text of the initial error reason, followed by text linking to a support ticket.

## Preview 📷

https://user-images.githubusercontent.com/114685994/194411756-d3fb1fc0-50b2-49e1-a7d2-c48f5c76759f.mov

## How to test 🧪

**What are the steps to reproduce the issue or verify the changes?**
1. Set the account Thing Limit to 0.
2. Attempt to create a new linode (or firewall, nodebalancer, volume) by filling in the required information.
3. Click 'Create' and the error message "Account Limit reached. Please open a support ticket." should be visible in the notice at the top of the page.
4. Click the linked text in the notice and confirm you are redirected to the support page with an open support ticket modal.

Applies to testing linkable text across multiple error messages (to be included in follow up issue):
5. Reset Thing Limit back to original value. Set Reputation as 0.
6. Attempt to create a new linode of any size.
7. Check that an error message stating "Additional verification is required to add this service. Please open a support ticket" appears at the top of the screen.
8. Click the linked text in the notice and confirm you are redirected to the support page with an open support ticket modal.
